### ISSUE TITLE
I get no results whatsoever; web browser searches return error 400

### ISSUE THREAD

**Issue by Xebtria** [PoE-TradeMacro/POE-TradeMacro#89]
For example:

http://i.imgur.com/S8fUQVs.png
same search in webbrowser:
http://poe.trade/search/asuionahowomob => 
```
Bad Request

The browser (or proxy) sent a request that this server could not understand.
```
poe.trade itself  seems to work (e.g. http://poe.trade/search/imaririteyoger works for me). I only have edited the additionalmacros txt with some other non-trade-related macros which I have. I also get no other errors from the ahk itself, all seems to work

what am I doing wrong?

(edit: using 1.1.6.1)
(edit2: same problem with 1.1.5.2/1.1.6, but it works with 1.1.5.1, so the problem seems to be introduced with 1.1.5.2/1.1.6; the same item from above creates this search in 1.1.5.1: http://poe.trade/search/ubetozubimokik)

**Comment by Genotron** [comment]
Thanks for the issue. Downloaded the macro the first time and couldn't find what/ if I was doing sth wrong.

1.1.5.1 works for me. the newer ones don't

**Comment by Eruyome** [comment]
I don't have this problem , but the last time it was encountered the wrong league names were used (races). Could you test this version? https://dl.dropboxusercontent.com/u/13620316/POE-TradeMacro.rar
Debug option is enabled and shows a message box with the leage name used in the search when searching.

**Comment by Eruyome** [comment]
Please check if v1.1.7 works for you.

**Comment by 51473R** [comment]
Hi, i've been having the same problem, updating to 1.1.7 still did not fix the issue.

**Comment by Eruyome** [comment]
Can you use the doprbox version I linked and give me feedback (league name)?

**Comment by 51473R** [comment]
Just downloaded your version and it seems to be working fine, but every time i use it it brings up a console labeled "ConsolDebugID" with the current time then essence. btw in in ESC

**Comment by Eruyome** [comment]
Yeah, that happens when the debug option is enabled. But when this version works, the other should too. Can you test both at the same time (almost the same, not together), someone mentioned that poe.trade was acting up sometimes. We have to m ake sure it's actually a problem with the macro.

**Comment by 51473R** [comment]
Ok, so i just tried running the "Run_without_admin_Elevation.ahk" instead of "Run_Only_This.ahk" and it worked so i closed it and went back to "Run_Only_This.ahk" and it now works perfectly on v1.1.7

**Comment by Eruyome** [comment]
So, it seems to be an issue with poe.trade sometimes acting up, but the macro works in general?

**Comment by 51473R** [comment]
Yeah its working fine now, its the first problem ive had with the macro, poe.trade isnt always reliable so thats probably the issue

**Comment by Eruyome** [comment]
There was also an issue with the league parsing, it didn't work the first time on a fresh install. This is fixed with 1.1.7.1
I hope all problems are fixed with this.

**Comment by Xebtria** [comment]
tried yesterday evening, works now with 1.1.7.1, thanks ! :)
